src/beamlime/services/fake_detectors.py

```diff
@@ -50,6 +50,53 @@
     'nmx': {
         f'detector_panel_{i}': (i * 1280**2 + 1, (i + 1) * 1280**2) for i in range(3)
     },
+    'bifrost': {
+        '123_channel_1_1_triplet': (1, 300),
+        '127_channel_1_2_triplet': (301, 600),
+        '131_channel_1_3_triplet': (601, 900),
+        '135_channel_1_4_triplet': (901, 1200),
+        '139_channel_1_5_triplet': (1201, 1500),
+        '144_channel_2_1_triplet': (1501, 1800),
+        '148_channel_2_2_triplet': (1801, 2100),
```

Comment:
Seems like there is a pattern, can we avoid spelling it out and use a loop or comprehension, to facilitate future changes?